packages/viewer/src/lib/scene-items/controller.ts

```diff
@@ -0,0 +1,98 @@
+import { Pager } from '@vertexvis/scene-tree-protos/core/protos/paging_pb';
+import { Uuid2l } from '@vertexvis/scene-tree-protos/core/protos/uuid_pb';
+import { PropertyCategoryMap } from '@vertexvis/scene-view-protos/sceneview/protos/domain_pb';
+import {
+  ListSceneItemMetadataRequest,
+  ListSceneItemMetadataResponse,
+} from '@vertexvis/scene-view-protos/sceneview/protos/scene_view_api_pb';
+import { SceneViewAPIClient } from '@vertexvis/scene-view-protos/sceneview/protos/scene_view_api_pb_service';
+import { UUID } from '@vertexvis/utils';
+import { Timestamp } from 'google-protobuf/google/protobuf/timestamp_pb';
+
+import { createMetadata, JwtProvider, requestUnary } from '../grpc';
+import { PagingLinks } from '../types';
+import { mapListSceneItemMetadataResponseOrThrow } from './mapper';
+
+export interface DomainPropertyKey {
+  name: string;
+  category: PropertyCategoryMap[keyof PropertyCategoryMap];
+}
+
+export interface TimestampDomainPropertyValue {
+  type: 'timestamp';
+  value: Timestamp.AsObject;
+}
+
+export interface StringDomainPropertyValue {
+  type: 'string';
+  value: string;
+}
+
+export interface LongDomainPropertyValue {
+  type: 'long';
+  value: number;
+}
+
+export interface DoubleDomainPropertyValue {
+  type: 'double';
+  value: number;
+}
+
+export type DomainPropertyValue =
+  | StringDomainPropertyValue
+  | LongDomainPropertyValue
+  | DoubleDomainPropertyValue
+  | TimestampDomainPropertyValue;
+
+export interface DomainPropertyEntry {
+  id: string;
+  key?: DomainPropertyKey | null;
+}
+export interface SceneItemMetadataResponse {
+  paging: PagingLinks;
+  entries: DomainPropertyEntry[];
+}
+
+export interface ListSceneItemMetadataOptions {
+  cursor?: string;
+  size?: number;
+}
```

Comment:
nit: could we potentially pull these types out into a separate `types.ts` file? I'm anticipating this list will only grow once we add the direct SceneItem endpoint, so it'd be nice to keep those definitions separate from the controller